src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -120,208 +105,198 @@ export const MyProfile = () => {
                 sx={{
                   display: 'grid',
                   gridTemplateColumns: { xs: '1fr', lg: '2fr 1fr' },
+                  columnGap: '0.5rem',
+                  rowGap: '0.5rem',
                   gridTemplateAreas: {
                     xs: '"profile-picture" "personalia-details"',
                     lg: '"personalia-details profile-picture"',
                   },
                   m: '1rem',
                 }}>
                 <Grid container rowGap={1} columns={16} sx={{ gridArea: 'personalia-details' }}>
-                  <Grid item xs={16} md={3}>
-                    <StyledTypography>{t('my_page.my_profile.person_name')}</StyledTypography>
+                  <Grid item sx={{ width: '100%' }}>
```

Comment:
For å holde det konsekvent på alle `<Grid item />`
```suggestion
                  <Grid item xs={16}>
```